### ISSUE TITLE
Ruler and Marks are off to the right on game screen

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/balancing-act#156]
**Test device**
Samsung
**Operating System**
Win 11
**Browser**
Chrome
**Problem description**
For https://github.com/phetsims/qa/issues/1073
On the game screen, not on the other screens from what I've seen, the ruler and marks are off to the right of their intended position on the balance. I don't see this in published.

**Visuals**
![offmark2](https://github.com/phetsims/balancing-act/assets/41024075/f27df59c-7b8d-45fb-bace-93d8673814b1)
![offcenter](https://github.com/phetsims/balancing-act/assets/41024075/663f925e-ee77-4da7-b4c7-054e94238686)

Troubleshooting information:

<details>

!!!!! DO NOT EDIT !!!!!
Name: ‪Balancing Act‬
URL: https://phet-dev.colorado.edu/html/balancing-act/1.3.0-dev.1/phet/balancing-act_all_phet.html
Version: 1.3.0-dev.1 2024-04-22 16:12:40 UTC
Features missing: applicationcache, applicationcache, touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/123.0.0.0 Safari/537.36
Language: en-US
Window: 1536x695
Pixel Ratio: 1.25/1
WebGL: WebGL 1.0 (OpenGL ES 2.0 Chromium)
GLSL: WebGL GLSL ES 1.0 (OpenGL ES GLSL ES 1.0 Chromium)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 31 uniform: 4096
Texture: size: 8192 imageUnits: 32 (vertex: 32, combined: 64)
Max viewport: 8192x8192
OES_texture_float: true
Dependencies JSON: {}

</details>

**Comment by Luisav1** [comment]
In the other screens, the default column state is `DOUBLE_COLUMNS`, but in the Game screen, it was `SINGLE_COLUMN`.

During challenges, `setChallenge()` temporarily sets the column state to `DOUBLE_COLUMNS` to prevent  inaccuracies, then reverts it back to the correct state for the current challenge.

To fix the offset issue, I set the default Game screen column state to `DOUBLE_COLUMNS`, which resolved it. I'm not entirely sure why switching from `SINGLE_COLUMN` to `DOUBLE_COLUMNS` was causing an offset, but at least it's working now.

I'm concerned about potential issues with changing the default state since there are many functions and a few links related to `columnStateProperty`, but they all seem safe. Additionally, I've tested it a bit, and everything appears to be fine.

Here's what it looks like now:
![image](https://github.com/phetsims/balancing-act/assets/52978048/5baff91f-1a1e-4d89-bb8a-134d599629f1)

Still, I'd like to keep this open for QA review to ensure it's working good and that there are no side effects.
